src/components/ItaliaTheme/MegaMenu/MegaMenu.jsx

```diff
@@ -112,17 +112,18 @@ const MegaMenu = ({ item, pathname }) => {
 
   if (item.mode === 'simpleLink') {
     return item.linkUrl?.length > 0 ? (
-      <NavItem tag="li">
+      <NavItem tag="li" active={isItemActive} role="none">
         <NavLink
           href={item.linkUrl === '' ? '/' : null}
           item={item.linkUrl[0]?.['@id'] ? item.linkUrl[0] : '#'}
           tag={UniversalLink}
           active={isItemActive}
           data-element={item.id_lighthouse}
+          role="menuitem"
         >
           <span dangerouslySetInnerHTML={{ __html: item.title }}></span>
           {isItemActive && (
-            <span className="visually-hidden">
+            <span className="sr-only">
```

Comment:
Questo lascialo `visually-hidden`. Fa la stessa cosa, ma visually-hidden è il nome nuovo della classe sr-only in v3 (che poi dipende da bootstrap e non da noi)